public/editor/scripts/editor/js/fc/project-rename.js

```diff
@@ -117,28 +117,30 @@ define(function(require) {
   }
 
   function save(context) {
-    context.saveButton.text("{{ renameProjectSavingIndicator }}");
-
-    persist.call(context, context.titleBar.val(), function(err) {
-      if(err) {
-        console.error("[Bramble] Failed to rename the project with ", err);
-        return;
-      }
+    // Do not apply save if project title remains the same
+    if (context.titleBar.val() === Project.getTitle()) {
```

Comment:
we can lift this out as a variable `var contextTitle = context.titleBar.val();` and then we can use that `contextTitle` here, as well as on line 126 and 132